src/main/java/org/eclipse/xpanse/terraform/boot/api/TerraformApiController.java

```diff
@@ -55,12 +54,13 @@ public TerraformApiController(TerraformExecutor terraformExecutor) {
      */
     @Tag(name = "Terraform", description = "APIs for running Terraform commands")
     @Operation(description = "Check health of Terraform API service")
-    @GetMapping(value = "/health", produces = MediaType.APPLICATION_JSON_VALUE)
+    @GetMapping(value = "/health/{module_directory}", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
@jinyangyang222 we must remove this parameter from the URI. 
instead generate one folder with random name (it can be a UUID too) under 'terraform.root.module.directory' directory and then execute the test script.

Reply:
fixed